requirements.in

```diff
@@ -16,3 +16,4 @@ uwsgi
 djangorestframework
 django-extensions
 ocw-data-parser==0.1.6
+moto
```

Comment:
Put this in `test_requirements.txt` instead

Reply:
Moved